### PR TITLE
add recipe for svim-asm

### PR DESCRIPTION
**Pull request by eldariont**
This pull request adds a bioconda recipe for svim-asm. svim-asm is a caller for large genomic variants from genome assemblies.

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by eldariont** [comment]
@BiocondaBot please add label

File: recipes/svim-asm/build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```
Comment (raivivek) [inline recipes/svim-asm/build.sh]: Probably just use `pip` to install this package? Please see the other comment.

File: recipes/svim-asm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  name: svim-asm
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/eldariont/svim-asm/archive/v{{ version }}.tar.gz
+  sha256: 380984261ead51b400203c0072f3c206c2d74c01ada96e3eec3d10e85d0d7a44
+
+build:
+  number: 0
+  noarch: python
```
Comment (raivivek) [inline recipes/svim-asm/meta.yaml]: `script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"`

**Comment by raivivek** [comment]
@bioconda-bot please merge

**Pull request merged**